src/plugins/embeddable/public/lib/panel/panel_header/panel_header.tsx

```diff
@@ -58,6 +58,21 @@ function renderBadges(badges: Array<Action<EmbeddableContext>>, embeddable: IEmb
   ));
 }
 
+function renderNotifications(
+  notifications: Array<Action<EmbeddableContext>>,
+  embeddable: IEmbeddable
+) {
+  return notifications.map(notification => (
+    <EuiNotificationBadge
+      key={notification.id}
+      style={{ marginTop: '4px', marginRight: '4px' }}
+      onClick={() => notification.execute({ embeddable })}
```

Comment:
is this doing anything?

Reply:
It calls action's `.execute()` method on click. We currently are not using it&mdash;the `.execute()` method of our action that we attach to `PANEL_NOTIFICATION_TRIGGER` is a noop.

So, in theory, right now, I could remove this line. But I've left it in case some action that attaches to `PANEL_NOTIFICATION_TRIGGER` actually has `.execute()` method implemented. Similar to have it is done for the `PANEL_BADGE_TRIGGER`.